src/RpcClient.ts

```diff
@@ -509,15 +508,13 @@ export class RpcClient {
    * @param {TransactionInstruction[]} instructions - The transaction instructions
    * @param {Signer[]} signers - The transaction's signers. The first signer should be the fee payer
    * @param {AddressLookupTableAccount[]} lookupTables - The lookup tables to be included in a versioned transaction. Defaults to `[]`
-   * @param {SendOptions} sendOptions - Options for sending the transaction. Defaults to `{ skipPreflight: false }`
-   * @returns {Promise<TransactionSignature>} - The transaction signature
+   * @returns {Promise<SmartTransactionContext>} - The transaction with blockhash, blockheight and slot
   */
   async createSmartTransaction(
     instructions: TransactionInstruction[],
     signers: Signer[],
     lookupTables: AddressLookupTableAccount[] = [],
-    sendOptions: SendOptions = { skipPreflight: false },
```

Comment:
@0xIchigo I don't think we need `sendOptions` for `createSmartTransaction`. Return type does not also match.